README.md

```diff
@@ -432,6 +442,9 @@ using a combination of `bwa` and `samtools` to output a BAM file instead of a SA
 #### Module parameters
 
 - A module file SHOULD only define input and output files as command-line parameters to be executed within the process.
+    - All possible input files MUST have a defined input channel declaration (see [Input/output options](#input-output-options) for dealing with optional input files)
+    - Directly associated auxiliary files to an input file CAN be defined within the same input channel alongside the main input channel  (e.g. [BAM and BAI](https://github.com/nf-core/modules/blob/e937c7950af70930d1f34bb961403d9d2aa81c7d/modules/samtools/flagstat/main.nf#L22)).
+    - Other generic auxiliary files used across different input files (e.g. common reference sequences) CAN be defined using a dedicated input channel (e.g. [reference files](https://github.com/nf-core/modules/blob/3cabc95d0ed8a5a4e07b8f9b1d1f7ff9a70f61e1/modules/bwa/mem/main.nf#L21-L23))
```

Comment:
Nice!

Reply:
Think this should go in the `Input/output options` section though as it relates to handling those features of the module.

Reply:
`Module parameters` are for things like `options.args` / command-line parameters used to run the tool